src/test/functional/shared/apiHelpers.ts

```diff
@@ -94,6 +130,24 @@ export const suspendUser = async (userId) => {
   }
 };
 
+export const convertISODateTimeToUTCFormat = (date: string): string => {
```

Comment:
Is there no way you can call the actual method in the "section" section? This looks identical to that

Reply:
All methods in test suite should be independent of main section, if possible

Reply:
Imported method from main